### PR TITLE
Fix getNotes

### PR DESCRIPTION
**Pull request by lotoussa**
to avoid TypeError: Cannot read properties of undefined (reading '$$eval')

### CODE REVIEW

**Comment by lotoussa** [comment]
Need to remove the '(' line 37
  const cleared = (getNotesTc.length === 0

File: js/tests/keycodes-symphony-dom_test.mjs [hunk 1]
```diff
@@ -24,15 +20,21 @@ tests.push(async ({ page, eq, ctx }) => {
...
 })
 
 tests.push(async ({ page, eq, ctx }) => {
   // check that all the notes are cleared when Escape key is pressed
   await page.keyboard.down('Escape')
-  const cleared = (await ctx.getNotes()).length === 0
+  const getNotesTc = await page.$$eval('.note', (nodes) => {
+    return nodes.map((note) => note.textContent)
+  })
+  const cleared = (getNotesTc.length === 0
```
Comment (nprimo) [inline js/tests/keycodes-symphony-dom_test.mjs:37]: as you pointed out, this line is missing a parenthesis

File: js/tests/keycodes-symphony-dom_test.mjs [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
-
 const characters = `didyouhandlethekeydowneventcorrectly`
 
 tests.push(async ({ page, eq, ctx }) => {
   // check that a note is created and matches the right letter when a key is pressed
   for (const [i, character] of characters.split('').entries()) {
     await page.keyboard.down(character)
     const typed = characters.slice(0, i + 1).split('')
-    eq(await ctx.getNotes(), typed)
+    const getNotesTc = await page.$$eval('.note', (nodes) => {
```
Comment (nprimo) [inline js/tests/keycodes-symphony-dom_test.mjs:10]: is there any special reason why the new function is `getNotesTc` instead of `getNotes`?

**Review by nprimo — CHANGES_REQUESTED** [review]
I have left a couple of inline comments; beside that I would have declared the new `getNotesTc` (or simply `getNotes`) in the main scope to avoid repetition

File: js/tests/keycodes-symphony-dom_test.mjs (hunk 2 above)
Comment (lotoussa) [reply js/tests/keycodes-symphony-dom_test.mjs:10]: Tc = TextContent (note.TextContent)

**Comment by lotoussa** [comment]
closed, nprimo exported this pr #1559

**Pull request closed without merging**